### ISSUE TITLE
Aspect ratio of images is not preserved

### ISSUE THREAD

**Issue by ragesoss** [videowikips/videowiki#25]
The aspect ratio of images that are used in a VideoWiki depend on the width of the viewport, and can be severely distorted.

Images within a video should maintain their original aspect ratio.

![narrow](https://user-images.githubusercontent.com/848483/55356035-bcb37200-547e-11e9-84ee-6e573e706ed0.png)
![wide](https://user-images.githubusercontent.com/848483/55356056-ca68f780-547e-11e9-8977-f8d0deaf77ad.png)

**Comment by videowikips** [comment]
Yes, aspect ratios are currently distorted. We will be fixing it hopefully by the End of Month. This bug is mentioned at https://en.wikipedia.org/wiki/Wikipedia:VideoWiki/Bug_report

**Comment by videowikips** [comment]
@ragesoss: Aspects ratios are fixed. Just some minor iterations are left, but drastically improved from the previous interface. 

Also the aspect of the final video is more or less fixed - You can check the VideoWiki vide on Dengue at :https://en.wikipedia.org/wiki/Dengue_fever

Let me know if you have any thoughts/inputs/feedback
